### PR TITLE
Added Unixtime conversion service

### PR DESCRIPTION
**Pull request by rodenacker**
Free REST API to convert datetime stamps to Unixtime and Unixtime to datetime

Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [ ] Your submission has a useful description
- [x] The description does not end with punctuation
- [ ] Each table column should be padded with one space on either side
- [ ] You have searched the repository for any relevant issues or pull requests
- [ ] Any category you are creating has the minimum requirement of 3 items
- [ ] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -284,6 +284,7 @@ API | Description | Auth | HTTPS | CORS |
 | [ScreenshotAPI.net](https://screenshotapi.net/) | Create pixel-perfect website screenshots | `apiKey` | Yes | Yes |
 | [SHOUTCLOUD](http://shoutcloud.io/) | ALL-CAPS AS A SERVICE | No | No | Unknown |
 | [StackExchange](https://api.stackexchange.com/) | Q&A forum for developers | `OAuth` | Yes | Unknown |
+| [Unixtime](http://unixtime.co.za/) | Convert to and from Unixtime | No | Yes | Unknown |
```
Comment (marekdano) [inline README.md:287]: Can you please change `http` to `https`?

**Review by marekdano — CHANGES_REQUESTED** [review]
Thanks for the contribution @rodenacker!

Can you please update your record (comment below) and also resolve the conflict?

**Comment by rodenacker** [comment]
This is all helluva confusing. Have I done it right?
Franz

**Comment by marekdano** [comment]
> This is all helluva confusing. Have I done it right?
> Franz

Yes, you did it @rodenacker! Thanks for it. 
In the future, if you're not sure about your changes go to [the `Files changed` tab ](https://github.com/public-apis/public-apis/pull/1361/files) to find out what you did.

**Review by marekdano — APPROVED** [review]

**Pull request merged**